README.md

```diff
@@ -0,0 +1,101 @@
+# OpenEuropa oe_oembed
+
+The OpenEuropa oe_oembed provides ...`[One Paragraph of project description goes here]`
```

Comment:
Should be updated with meaningful information.

Reply:
Yes, thanks!